tests/Integration/Lti/Responder/LtiOutcomeResponderTest.php

```diff
@@ -57,4 +54,30 @@ public function testCreateReplaceResultResponse(): void
 
         self::assertSame($this->getValidReplaceResultResponseXml($messageIdentifier), $response->getContent());
     }
+
+    private function getValidReplaceResultResponseXml(string $messageIdentifier): string
+    {
+        return <<<XML
```

Comment:
suggestion: instead of hardcoding the xml in your test, what about using the [available template](https://github.com/oat-sa/lib-lti1p3-basic-outcome/blob/master/templates/response/replaceResultResponse.xml.twig) for this ? wdyt ?

Reply:
I think it's better to not rely on vendor party templates in tests.

Reply:
see https://github.com/oat-sa/simple-roster/pull/238#discussion_r715564694